### ISSUE TITLE
Linux: Missing classes

### ISSUE THREAD

**Issue by pygy** [moai/moai-dev#430]
The following classes are absent of Linux builds. It's related to the platform-specific nature of Lua bytecode. A Lua compile pass should probably be added to the build process... or skip bytecode altogether and load from source rather than bytecode.

See this thread:http://getmoai.com/forums/post5583.html#p5583
- MOAICamera2D
- MOAIHttpTask
- MOAILayer2D
- MOAILayerBridge2D
- MOAIProp2D
- MOAITransform2D

Basically all functions defined here: https://github.com/moai/moai-dev/blob/linux/src/lua-headers/moai.lua
